### DISCUSSION TITLE
My task

### DISCUSSION THREAD

**Discussion by crashyyy** [codewars/codewars.com#3097] [General]
For a given two-dimensional array of numbers 0 or 1 of size MxN, find the size of the largest subarray composed of only 0s.

Example.
Entry:
1 0 1 1 0
1 0 1 0 0
1 0 0 0 0
1 0 1 0 1
Output: 1x4, 2x2 and 4x1

Any ideas how to create algorithm? 

PS. Unfortunately, my current skills only allow me to find a one-dimensional subarray like 1x4 I dont know how to find 2x2, so I am looking for some help.

**Comment by hobovsky** [comment]
Codewars Github is not meant for help with programming tasks. It's meant to discuss matters related to Codewars platform, the website, its functions, etc.

For help with your assignment please visit your favorite homework forum, or you can try your luck asking for help on Codewars Discord.

Closing as off-topic.
